Directory.Build.targets

```diff
@@ -59,100 +59,124 @@
 
   <!-- For build debugging purposes. -->
   <Target Name="Vars">
+
+    <ItemGroup>
+      <GeneralItems Include="
```

Comment:
Nit since MsBuild's items and properties are like global variables, perhaps add a common prefix to these wiht an underscore to indicate they are private variables and folks should not mess with them...

i.e. `_variablesToPrintForDebugGeneral`.

I wished msbuild had some proper form of scoping variables to a target, or at least a targets file... But every variable is like a global public read/write variable :( and

Reply:
Ah, this is unfortunate. I had hoped defining the items inside the target would scope them.

I'll rename accordingly.